api_spec.yaml

```diff
@@ -5134,16 +5150,95 @@ paths:
                     type: boolean
         '400':
           $ref: '#/components/responses/400'
-        
+
+        '401':
+          $ref: '#/components/responses/401'
+
+        '402':
+          $ref: '#/components/responses/402'
+
+        '404':
+          $ref: '#/components/responses/404'
+
+        '500':
+          $ref: '#/components/responses/500'
+
+  /recipe/multitenancy/connectionuridomain/v2:
+    put:
+      tags:
+        - Multitenancy Recipe
+
+      description: |
+        Create or update a connection uri domain.
+
+        SuperTokens subscription license key is required, if not using the base connection uri domain.
+
+        This request must originate from public app and public tenant on the base
+        connection uri domain.
+
+        This `firstFactors` can be set to following values:
+
+        - `null`: When set to `null`, the SDK will use firstFactors defined in the SDK
+        - `[]` (empty array): No first factors would be enabled for the tenant
+        - non-empty array: The first factors that are enabled for the tenant
+
+        If first factors are not specified while creating a new connection uri domain, all the login methods will be enabled by default.
+
+        `requiredSecondaryFactors` can be set to null to remove all entries in the core, or a non empty string array to be updated in the core. Setting of empty array is disallowed.
```

Comment:
right. But what is the effect or removing it and setting it in the core? Please be clear. And make it a bullet point like you have done for firstFactors